### ISSUE TITLE
you know what you should do 

### ISSUE THREAD

**Issue by gummybear17** [AndreiRudenko/linc_luajit#12]
fork the linc_lua and maybe you can fix it

**Comment by AndreiRudenko** [comment]
Im not maintaining linc_lua repo, ask Kevin please. 
And also please create issues only for this repository issues, and with names, please use descriptive issue names, and dont flood with many messages in issue. 
Thank you!

**Comment by gummybear17** [comment]
oh ok cause idk where to contact you in

**Comment by gummybear17** [comment]
can you give me the link?

**Comment by AndreiRudenko** [comment]
Again, please dont flood with many messages...
I already post links in different issue, but here you go.
https://github.com/kevinresol/linc_lua
https://github.com/kevinresol/hxvm-lua

**Comment by gummybear17** [comment]
ok sorry

**Comment by AndreiRudenko** [comment]
No worries, maybe it will be helpful to to know how to use github.
For chatting some repositories have Discussions.
And Issues is for... issues :)

**Comment by gummybear17** [comment]
can you have discussion enable in the settings of the repository?

**Comment by AndreiRudenko** [comment]
No, sorry. Having only issues is perfectly fine for this small lib.

**Comment by gummybear17** [comment]
oh ok i just think it whould be better
